### DISCUSSION TITLE
dumb question--how to add existing image to outpaint canvas?

### DISCUSSION THREAD

**Discussion by ride5k** [zero01101/openOutpaint#282] [Q&A] [answered]
i feel pretty silly asking, but i can't find it in the docs or by clicking around...

**Comment by dynatron832** [comment, marked as answer]
on the tools panel you have a "stamp" tool. it´s the seccond one from below and looks like a file icon with an arrow up on it. then on the left options you get an empty box. there you have two option´s to get your image as a stamp inside. #1 chose the "upload image" button. #2 drag and drop your image into the box. after the image is loaded you can place/stamp a copy of your image in the canvas. with the mouse wheel or the slider you can adjust the scale of your stamp image.
